include/circt/Dialect/FIRRTL/AnnotationDetails.h

```diff
@@ -28,6 +28,8 @@ constexpr const char *rawAnnotations = "rawAnnotations";
 // Annotation Class Names
 //===----------------------------------------------------------------------===//
 
+constexpr const char *conventionAnnoClass =
+    "firrtl.linkage.Convention";
```

Comment:
@seldridge @jackkoenig I'm adding this annotation to mfc before adding it to chisel. Any suggestions on a good class name?